react/CozyDialogs/Dialog.jsx

```diff
@@ -35,6 +36,7 @@ const Dialog = ({
       fullScreen={isFullscreen}
       classes={{ paper: paperClassName }}
       aria-labelledby={`modal-title-${id}`}
+      {...otherProps}
```

Comment:
classes is overriden here ? I am not sure that it is what's expected by the user. I think otherProps.classes should be merged.

Reply:
Same thing for TransitionProps

Reply:
Nice catch 👍

Reply:
Could you modify the custom Dialog example with the utilisation of these features ?

Reply:
I've done it in https://github.com/cozy/cozy-ui/pull/1640